calicoctl/commands/patch.go

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) 2016-2019 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/docopt/docopt-go"
+	"github.com/projectcalico/calicoctl/calicoctl/commands/constants"
+	log "github.com/sirupsen/logrus"
+)
+
+func Patch(args []string) error {
+	// TODO: write coherent description.
```

Comment:
Is this TODO still valid?

Reply:
Ah, yes, I suppose it was more of a reminder to get it proofread by someone else. So if you're happy, I'll remove the TODO

Reply:
Yeah, this looks fine to me!